libraries/.foodcritic

```diff
@@ -0,0 +1,4 @@
+~FC011
```

Comment:
I can't get it to **not** see the libraries directory as a separate cookbook, which is why I put that `.foodcritic` file in there. How are you able to run it so it gets ignored?